config/defaults.config.php

```diff
@@ -209,9 +209,9 @@
 		// If activated, all hashtags will point to the local server.
 		'local_tags' => false,
 
-		// logger_adapter (String)
-		// Sets the logging adapter of Friendica globally (monolog, syslog)
-		'logger_adapter' => 'monolog',
+		// logger_config (String)
+		// Sets the logging adapter of Friendica globally (monolog, syslog, stream)
+		'logger_config' => 'stream',
```

Comment:
Why id you rename the config key?

Reply:
Because `adapter` was the wrong word for it.. We had `cache_config` and `lock_config`. Adapter was just right in case of `config_adater`, because it's the adapter for the configuration (extending an existing functionality).

Reply:
Please keep in mind that there are people out there who used the old setting key. So there should be some transformation in any way for this.

Reply:
The `logger_adapter` was introduced with the last PR at develop, so there's little chance that there are existing configurations.